recipe/meta.yaml

```
@@ -9,8 +14,8 @@ source:
   sha256: 43616529f9b7d1afc83386f943dc66c4da5e052f00217ba7e3ad8dd1b5f3a825
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  string: "{{ build_string }}py{{ py }}h{{ PKG_HASH }}_{{ build }}"
+  number: {{ build }}
```

To weigh down the mkl build
```suggestion
  number: {{ build }}
  track_features:
    - numexpr_mkl    # [linux64 and mkl_support == 'mkl']
```